FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -26,17 +26,52 @@ public final class CoreDataFeedStore: FeedStore {
 			url: storeURL
 		)
 		context = container.newBackgroundContext()
+		context.undoManager = UndoManager()
 	}
+}
 
+extension CoreDataFeedStore: FeedStore {
 	public func retrieve(completion: @escaping RetrievalCompletion) {
-		fatalError("Must be implemented")
+		let context = self.context
+		context.perform {
```

Comment:
Suggestion: another way of capturing `context` to use inside this closure is by using a Capture List:
```suggestion
		context.perform { [context] in
```
If you decide to follow this suggestion, apply it to the other `context.perform` calls in this file to keep the code consistent.